src/main/java/inu/parser/Parser.java

```diff
@@ -0,0 +1,216 @@
+package inu.parser;
+
+import inu.exceptionhandling.EmptyTaskListException;
+import inu.exceptionhandling.EmptyUserInputException;
+import inu.exceptionhandling.ExceptionManager;
+import inu.task.DeadLine;
+import inu.task.Event;
+import inu.task.Task;
+import inu.task.Todo;
+import inu.task.TaskList;
+import inu.commons.Ui;
+import inu.commons.Util;
+
+
+public class Parser {
```

Comment:
Parser class is very long, and seem to do more than just 'parsing'. Consider having command classes if needed.